packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -66,6 +112,23 @@ abstract class ProgressIndicator extends StatefulWidget {
     super.debugFillProperties(properties);
     properties.add(PercentProperty('value', value, showName: false, ifNull: '<indeterminate>'));
   }
+
+  Widget _buildSemanticsWrapper({
+    @required BuildContext context,
+    @required Widget child,
+  }) {
+    String expandedSemanticsValue = semanticsValue;
+    if (semanticsLabel != null && value != null) {
```

Comment:
Why guard this on the presence of semanticsLabel? Wouldn't this be useful information for a11y users even if the developer foolishly forgo to set a semanticsLabel?

If devs really don't want this, they can just wrap the entire widget in a ExcludesSmeanitcs widget.

Reply:
Done